### DISCUSSION TITLE
Scores with unranked mods still show as calculating in-game

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#27026] [Q&A] [answered]
The display on the website for scores with unranked mods recently changed.

![image](https://github.com/ppy/osu/assets/74088315/f43812ad-b6ef-47c0-9474-0dd1fbd315cc)

However, when looking at this score in the in-game webview, it still says that it is calculating when it isn't.

![image](https://github.com/ppy/osu/assets/74088315/dc81a946-40a7-43e6-a1cb-755ae5cfcf08)

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/27027
